CHANGELOG

```diff
@@ -1,3 +1,7 @@
+* Wed Dec 04 2019 Trevor Vaughan <tvaughan@onyxpoint.com> - 4.0.1-0
+- Fixed an issue in `simplib::ip::family_hash` where the 'unknown' entries were
```

Comment:
Not really a bug fix.  You changed the function to handle hostnames (previously disallowed by parameter typing).

Reply:
@lnemsick-simp The reason that I think that it's a bugfix is that the `unknown` section of the hash was always supposed to be a catch-all. However, the way it was written, it couldn't actually function that way.